app/javascript/components/CreateSurvey.js

```diff
@@ -69,6 +80,60 @@ export default function CreateSurvey() {
         })
         .catch(console.log);
     };
+
+    const createTextQuestion = async (question) => {
+        const text_question = {
+            text_question: {
+                question: question.question, 
+                // order: i,
+                survey_id: surveyId
+            }
+        }
+
+        await fetch(`${baseUrl}/api/v1/text_questions/create`, {
+            method: 'POST',
+            body: JSON.stringify(text_question),
+            headers: {
+                'Content-Type': 'application/json'
+            },
+        })
+        .then(checkRequest)
+        .then(data => {
+            if  (data.error || data.notice) {
+                console.log("Failed to add question: " + (data.error||data.notice))
+            } else {
+                console.log("Question added: " + data.question);
+            }
+        })
+        .catch(console.log);
+    }
+
+    const createMCQuestion = async (question) => {
+        const text_question = {
```

Comment:
does this need to be changes later? I guess my McQuestion Component can't be merged with this front end huh.

Reply:
ahh i copied the code into a new function at the end and must've pasted the wrong thing, lmao one sec ill redo it

Reply:
fixed @AndrewHall797